### PR TITLE
PR for project proposal xw2600.md

### PR DESCRIPTION
**Pull request by rennahweng**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+# Contributing
```
Comment (timkpaine) [inline CONTRIBUTING.md]: don't need this file

File: README.md [hunk 2]
```diff
@@ -1,5 +1,33 @@
-# project-proposals-f2020
-Project Proposals for Fall 2020
+# Open Source Development
```
Comment (timkpaine) [inline README.md]: don't remove my readme

**Review by timkpaine — CHANGES_REQUESTED** [review]

**Comment by rennahweng** [comment]
I have resolved the changes I was requested to make and submitted hw2 (due 9/29) as well. Thank you!

File: xw2600.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# Open Source Development
+Link to my open source project repo: https://github.com/rennahweng/two-bias/tree/master
```
Comment (timkpaine) [inline xw2600.md]: shorten to just https://github.com/rennahweng/two-bias

**Review by timkpaine — CHANGES_REQUESTED** [review]

File: xw2600.md (hunk 3 above)
Comment (rennahweng) [reply xw2600.md]: Okie, changed!

**Pull request merged**